bootstrap-app/src/cmr/bootstrap/services/bootstrap_service.clj

```diff
@@ -62,22 +62,13 @@
   [context dispatcher date-time]
   (dispatch/index-data-later-than-date-time dispatcher context date-time))
 
-(defn- validate-collection
```

Comment:
I think we should keep this validation when trying to move a collection to a separate index (we don't want to create a new collection index for a non-existent collection).

Reply:
This is showing up in the Github diff as still deleted ...

Reply:
Its moved up in the file. I can fix that if you think its important.

Reply:
Hahaha, nah -- not important at all.